reference/library/src/morpheus-master/sass/base/_extendables.scss

```diff
@@ -204,10 +204,22 @@
 	}
 }
 
-.button, button.ui-state-default{
+.button, button.ui-state-default, .btn.btn-default {
 	@include sakai_button( $button-background-color, $button-border-color, $button-text-color, $button-background-secondary-color );
 }
 
+.button.link, a.btn-link, button.btn-link {
+	@include sakai_button( #FFF, #FFF, $primary-color, #FFF );
```

Comment:
Sure. Although, this particular kind of Bootstrap button is supposed to mimic an anchor, so the border and background will always be white. Unless there's a variable for anchor/link background color?